llvm/test/MC/Disassembler/Xtensa/code_density.txt

```diff
@@ -0,0 +1,64 @@
+# RUN: llvm-mc -triple=xtensa -mattr=+density -disassemble < %s | FileCheck -check-prefixes=CHECK-DENSITY %s
+# RUN: llvm-mc -triple=xtensa -disassemble %s &> %t
+# RUN: FileCheck -check-prefixes=CHECK-CORE < %t %s
+
+#------------------------------------------------------------------------------
+# Verify that binary code is correctly disassembled with
+# code density option enabled. Also verify that dissasembling without
+# density option generates warnings.
+#------------------------------------------------------------------------------
+
+0x4a 0x23
+# CHECK-DENSITY: add.n  a2, a3, a4
+# CHECK-CORE: [[#@LINE-2]]:1: warning: invalid instruction encoding
```

Comment:
The test fails at HEAD. It's recommended to use `[0x4a 0x23]` so that llvm-mc will disassemble the bytes as a whole and not fallback (skip one byte and continue disassembling), leading to weird diagnostics.

You could utilize the recent `--hex` so that you can do `[4a23]`

For llvm-mc, `s/< %s/%s/`

Reply:
@MaskRay, thank you very much for comment, I'm sorry, I tried to reproduce test fail using "check-llvm" but without success, may I ask you for advice about how to reproduce the problem?

Reply:
You can reproduce with just `ninja check-llvm-mc-disassembler-xtensa`. Is the test skipped in your build?

You could change `llvm/test/MC/Disassembler/Xtensa/lit.local.cfg` locally to see whether Xtensa is in config.root.targets

Reply:
I'm sorry, that was some misunderstanding on my part. I created PR https://github.com/llvm/llvm-project/pull/121073, could you PTAL?